hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/CryptoTransferHandler.java

```diff
@@ -239,7 +239,7 @@ public void handle(@NonNull final HandleContext context) throws HandleException
 
         // create a new transfer context that is specific only for this transaction
         final var transferContext =
-                new TransferContextImpl(context, enforceMonoServiceRestrictionsOnAutoCreationCustomFeePayments);
+                new TransferContextImpl(context, enforceMonoServiceRestrictionsOnAutoCreationCustomFeePayments, false);
```

Comment:
Please use an Enum instead of false. It will be more readable.

Reply:
I refactored it :)